contrib/mesos/pkg/node/node.go

```diff
@@ -36,11 +36,10 @@ const (
 )
 
 // Create creates a new node api object with the given hostname and labels
-func Create(client *client.Client, hostName string, labels map[string]string) (*api.Node, error) {
+func Create(client *client.Client, hostName string, slaveAttrLabels, annotations map[string]string) (*api.Node, error) {
```

Comment:
re 2nd question (zero values): good catch, that should indeed be unnecessary, will operate this out.